packages/elements/src/components/SchemaAndDescription/index.tsx

```diff
@@ -39,13 +38,7 @@ export const SchemaAndDescription = ({
       <JsonSchemaViewer
         resolveRef={resolveRef}
         className={cn(className, CLASSNAMES.block)}
-        schema={
-          {
-            ...schema,
-            title: undefined,
-            description: undefined,
```

Comment:
This was done on purpose as we didn't want to display title and description here. It could be just removed from original schema object, but we actually don't need to do that anymore so that change is fine